PYME/Acquire/Hardware/driftTracking.py

```diff
@@ -202,6 +202,38 @@ def set_focus_tolerance(self, tolerance):
     def get_focus_tolerance(self):
         return self.focusTolerance
 
+
+    def set_delta_Z(self, delta):

```

Comment:
Very minor note:

This is presumably just copying/ adapting the code for focus_tolerance, but I'm not sure we gain anything from having setter and getter functions over just setting the attribute directly (setters and getters are not really considered idiomatic in python). The best argument would probably be that it allows documentation of the parameters.